### PR TITLE
[wip] Kylel/cite preset hue

### PR DESCRIPTION
**Pull request by kyleclo**
- TODO: double check this runs all the way through to detect hue step

### CODE REVIEW

File: data-processing/README.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ If you're just testing out the pipeline, consider making an
 Then run the following command to start off the pipeline:
 
 ```bash
-python scripts/process.py fetch-arxiv-sources arxiv_ids.txt
+python scripts/process.py fetch-arxiv-sources --arxiv-ids-file arxiv_ids.txt
```
Comment (andrewhead) [inline data-processing/README.md]: Yup, good catch!

**Comment by kyleclo** [comment]
i've renamed the steps to have `visual-validate-` prefixes; adding more steps

**Comment by andrewhead** [comment]
Sounds good!

**Comment by andrewhead** [comment]
Hey @kyleclo ! Let's sync up to see what we want to do with this, perhaps next Tuesday. I wonder if I should close this PR if we're planning on approaching visual validation in another way, or if we should roll it in before this branch diverges to much from master.

**Comment by andrewhead** [comment]
Closing, as we have some very lightweight visual validation implemented (see #37), together with the ability to process just one entity at a time. We may return to this PR if we need more nuanced visual validation in the future.

**Pull request closed without merging**